### PR TITLE
Replacement of Can_Write function

### PR DESCRIPTION
**Pull request by shamakh96**
Can_Write function replaced

### CODE REVIEW

File: Software/bsw/static/Mcal/CAN/src/Can.c [hunk 1]
```diff
@@ -551,7 +551,7 @@ Std_ReturnType Can_SetBaudrate(uint8 Controller, uint16 BaudRateConfigID) {
 		/*
 		 * Baud rate Prescaler = tq * CAN Clock
 		 * bit time =  n * tq
-		 *          =[Sync + Prop + Phase1 + Phase2] � tq
+		 *          =[Sync + Prop + Phase1 + Phase2] � tq
```
Comment (autosarzs) [inline Software/bsw/static/Mcal/CAN/src/Can.c:554]: Is this mistake ?

File: Software/bsw/static/Mcal/CAN/src/Can.c [hunk 2]
```diff
@@ -775,152 +775,261 @@ void Can_MainFunction_BusOff(void) {
...
 	}
 
-	return returnVal;
+
 }
 
+
+
+
+
```
Comment (autosarzs) [inline Software/bsw/static/Mcal/CAN/src/Can.c:1032]: My question here, Is the team agreed of all this changes ? and why no one in the reviewer list ? :) 

Great work, keep it up, thanks for being the first one working in that :)

**Review by autosarzs — CHANGES_REQUESTED** [review]
just a small change

**Comment by autosarzs** [comment]
You don't need to open a new pull request if the only change is ( * ) above.

**Comment by shamakh96** [comment]
yes, it was a mistake.

yes, the team kind of agree :'D

ok, i'll fix somethings and inform you.

File: Software/bsw/static/Mcal/CAN/src/Can.c (hunk 1 above)
Comment (shamakh96) [reply Software/bsw/static/Mcal/CAN/src/Can.c:554]: yes, it was a mistake

File: Software/bsw/static/Mcal/CAN/src/Can.c (hunk 2 above)
Comment (shamakh96) [reply Software/bsw/static/Mcal/CAN/src/Can.c:1032]: yes, kind of :'D

**Pull request closed without merging**